manifest.php

```diff
@@ -29,26 +28,26 @@
 
 return array(
     'name' => 'taoPublishing',
-	'label' => 'Test Publishing',
-	'description' => 'An extension to publish tests to a delivery environment',
+    'label' => 'Test Publishing',
+    'description' => 'An extension to publish tests to a delivery environment',
     'license' => 'GPL-2.0',
-    'version' => '5.1.0',
+    'version' => '5.2.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => array(
-        'taoDeliveryRdf' => '>=13.1.0',
+        'taoDeliveryRdf' => '>=13.3.0',
         'tao' => '>=45.0.0',
         'taoQtiTest' => '>=38.13.0',
     ),
-	'managementRole' => 'http://www.tao.lu/Ontologies/generis.rdf#taoPublishingManager',
+    'managementRole' => 'http://www.tao.lu/Ontologies/generis.rdf#taoPublishingManager',
     'acl' => array(
-        array('grant', 'http://www.tao.lu/Ontologies/generis.rdf#taoPublishingManager', array('ext'=>'taoPublishing')),
+        array('grant', 'http://www.tao.lu/Ontologies/generis.rdf#taoPublishingManager', array('ext' => 'taoPublishing')),
     ),
     'install' => array(
         'rdf' => array(
-            __DIR__. '/model/ontology/platform.rdf',
-            __DIR__. '/model/ontology/taodelivery.rdf',
-            __DIR__. '/model/ontology/taotest.rdf',
-            __DIR__. '/model/ontology/indexation.rdf'
+            __DIR__ . '/model/ontology/platform.rdf',
```

Comment:
Missing add new installation script call

Reply:
added thank you

Reply:
done